### ISSUE TITLE
"Mark as read" button broken on any update above 94.0. 

### ISSUE THREAD

**Issue by WizardMascot** [eni9889/IG-PP-Issues#455]
You can "mark as read" once before you're forced to restart the application as it does not work any more after that. This is on any update after 94.0 and is really annoying, not going to lie.... A fix would be more than greatly appreciated

Also if you do release an update, could you tell us on the tweak page what update it 100% works with so it's easier for us to track if we're on the wrong version or not

Kind regards

**Comment by vagvalas** [comment]
Also here

**Comment by eni9889** [comment]
Is this on sideloaded?

**Comment by WizardMascot** [comment]
No, on jailbroken iPhone. If you do not have a jailbroken iPhone anymore, I don’t know. I'll more than gladly help

**Comment by vagvalas** [comment]
No, jailbroken, uncover 3.4.2
on 12.1.4
on 12.1.1
and on 12.2 tested

**Comment by vagvalas** [comment]
me also.

**Comment by eni9889** [comment]
Read receipts for stories or DM? @vagvalas @WizardMascot

**Comment by WizardMascot** [comment]
For DM's, stories work fine on all versions

**Comment by vagvalas** [comment]
The "Mark as seen" button.

**Comment by WizardMascot** [comment]
Hey, thanks for the update, but which version does it work with? Because it isn't working on the latest one..
